package.json

```diff
@@ -16,6 +16,8 @@
     "webpack": "1 || ^2.1.0-beta"
   },
   "devDependencies": {
+    "babel-core": "^6.0.0",
```

Comment:
Which is why it is a `peerDependency`. At what point would someone try and use this plugin without already having installed `babel-core` or had it come along with something else `babel` related?

We really shouldn't be packaging `babel-core` in the webpack loader imo.

If it's explicitly added - you're good
If it's a `dependency` of X - you're good
If for some strange reason neither of the above are true, you get a `peerDependency` warning and need to `npm i`